aiokafka/client.py

```diff
@@ -574,6 +575,7 @@ def _check_api_version_response(self, response):
         # in descending order. As soon as we find one that works, return it
         test_cases = [
             # format (<broker version>, <needed struct>)
+            ((2, 3, 0), SyncGroupRequest[0].API_KEY, 3),
```

Comment:
Those checks are meant to pick a working version, why would you introduce a 2nd check for version 2.3.0? What version introduces SyncGroup v3?

Reply:
It is introduced in version 2.3